Tweak installation reading flow guidance (#2447)
nf-core/website: src/content/docs/usage/installation.md

We recommend using a personal installation of Nextflow where possible, instead of using a system-wide installation.
This makes it easier to update and control versions.

We generally recommend following the official installation instructions from the Nextflow documentation.
However on this page we provide '[quick start](#quick-start-installation)' version of these instructions, as well as instructions for [installing via Conda](#bioconda-instllation), and on [Windows operating systems](#installation-on-windows).

> At any point, if double, see the [official Nextflow installation documentation](https://www.nextflow.io/docs/latest/getstarted.html#installation).

Reviewer: ```suggestion
If in doubt, see the [official Nextflow installation documentation](https://www.nextflow.io/docs/latest/getstarted.html#installation).
```
